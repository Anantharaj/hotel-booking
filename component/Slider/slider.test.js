import React from "react";
import renderer from "react-test-renderer";
import InputSlider from "./index";

it("Slider renders correctly", () => {
  const tree = renderer.create(<InputSlider handleChange={() => {}} sliderValue={30} min={0} max={100} />).toJSON();
  expect(tree).toMatchSnapshot();
});
