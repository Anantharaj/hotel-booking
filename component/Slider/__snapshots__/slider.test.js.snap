// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Slider renders correctly 1`] = `
<div
  className="MuiBox-root css-gajhq5"
>
  <p
    className="MuiTypography-root MuiTypography-body1 MuiTypography-gutterBottom css-d3wcwz-MuiTypography-root"
    id="input-slider"
  >
    Price Filter
  </p>
  <div
    className="MuiGrid-root MuiGrid-container MuiGrid-spacing-xs-2 css-ofrcno-MuiGrid-root"
  >
    <div
      className="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-true css-1vd824g-MuiGrid-root"
    >
      <span
        aria-labelledby="input-slider"
        className="MuiSlider-colorPrimary MuiSlider-sizeMedium MuiSlider-root css-187mznn-MuiSlider-root"
        onMouseDown={[Function]}
      >
        <span
          className="MuiSlider-rail css-14pt78w-MuiSlider-rail"
        />
        <span
          className="MuiSlider-track css-1gv0vcd-MuiSlider-track"
          style={
            Object {
              "left": "0%",
              "width": "30%",
            }
          }
        />
        <span
          className="MuiSlider-thumbColorPrimary MuiSlider-thumbSizeMedium MuiSlider-thumb css-eg0mwd-MuiSlider-thumb"
          data-index={0}
          onMouseLeave={[Function]}
          onMouseOver={[Function]}
          style={
            Object {
              "left": "30%",
              "pointerEvents": undefined,
            }
          }
        >
          <input
            aria-labelledby="input-slider"
            aria-orientation="horizontal"
            aria-valuemax={100}
            aria-valuemin={0}
            aria-valuenow={30}
            data-index={0}
            disabled={false}
            max={100}
            min={0}
            onBlur={[Function]}
            onChange={[Function]}
            onFocus={[Function]}
            step={1}
            style={
              Object {
                "border": 0,
                "clip": "rect(0 0 0 0)",
                "direction": "ltr",
                "height": "100%",
                "margin": -1,
                "overflow": "hidden",
                "padding": 0,
                "position": "absolute",
                "whiteSpace": "nowrap",
                "width": "100%",
              }
            }
            type="range"
            value={30}
          />
        </span>
      </span>
    </div>
    <div
      className="MuiGrid-root MuiGrid-item css-13i4rnv-MuiGrid-root"
    >
      <h2
        className="MuiTypography-root MuiTypography-subtitle1 css-kbbqzj-MuiTypography-root"
      >
        30
         PD
      </h2>
    </div>
  </div>
</div>
`;
